package.json

```diff
@@ -42,8 +42,10 @@
     "wreck": "5.5.1"
   },
   "devDependencies": {
-    "eslint": "3.0.0",
+    "addons-linter": "0.14.0",
+    "eslint": "2.13.1",
```

Comment:
Reduced this from 3.x to 2.x since **eslint@3** is for Node 4+. We should change this back to v3 if we upgrade to a newer version of Node.js (Ref: #1204).